src/common/common.js

```diff
@@ -403,6 +403,24 @@ const common = {
     }
     return '0';
   },
+  /**
+   * sortTokens
+   * @param {Array} tokens, array of objects {symbol, token}
+   * @returns array of sorted objects
+   */
+  sortTokens(tokens) {
+    return tokens.sort((a, b) => {
+      if (a.type !== b.type) {
+        return b.type === 'Testnet' ? -1 : 1;
+      }
+      let symbolIndexA = _.findIndex(supportedTokens, (token) => a.symbol === token);
+      // If token is not found in supportedTokens, indicating it is a custom token, Should be at the end of the list
+      symbolIndexA = symbolIndexA !== -1 ? symbolIndexA : Number.MAX_SAFE_INTEGER;
+      let symbolIndexB = _.findIndex(supportedTokens, (token) => b.symbol === token);
+      symbolIndexB = symbolIndexB !== -1 ? symbolIndexB : Number.MAX_SAFE_INTEGER;
+      return symbolIndexA - symbolIndexB;
```

Comment:
What if you get indexA === -1 and indexB === -1?

If the return is 0, does that mean sometimes you get A, B, and other times get B, A?

Reply:
The best solution is to sort they by time added to the list.

or you can sort by symbol name.

Reply:
Good question!

Reply:
Fixed